RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate InviteUsersEditView to TypeScript

## Proposed changes
Migrates `InviteUsersEditView` to TypeScript

## Issue(s)	
N/A

## How to test or reproduce
- You need to be an owner of a channel
- Then go to `Actions` and click on `Invite users`
- click at `Edit Invite`, then check if everything still works fine.

## Screenshots
N/A

## Types of changes




## Checklist



## Further comments
N/A
